pkg/oauth/user.go

```diff
@@ -207,16 +211,49 @@ func (s *server) Login(c echo.Context) error {
 	return c.NoContent(http.StatusNoContent)
 }
 
+var errInvalidLogoutRedirectURI = errors.DefinePermissionDenied("invalid_logout_redirect_uri", "the redirect URI did not match the one(s) defined in the client")
+var errMissingAccessTokenIDParam = errors.DefinePermissionDenied("missing_param_access_token_id", "Access token ID was not provided")
+
 func (s *server) Logout(c echo.Context) error {
 	ctx := c.Request().Context()
-	session, err := s.getSession(c)
+	accessTokenID := c.QueryParam("access_token_id")
+	if accessTokenID == "" {
+		return errMissingAccessTokenIDParam
```

Comment:
So `access_token_id` is required now? How would the user logout from their current (cookie) session on the OAuth server?

Reply:
The idea is that logging out will only work over our official SSO clients. Since the access token and session are connected, we don't need an isolated oauth server logout anymore.
Even if the access token is lost, our clients will automatically obtain a new access token that is connected to the current session in the OAuth server and can thus be used to logout (thus deleting the connected session on the OAuth server).